.prettierignore

```diff
@@ -1,4 +1,17 @@
 # The GH actions don't seem to compile and verify themselves well when Prettier is applied to them
 .github/actions/javascript/**/index.js
+.well-known
 desktop/dist/**/*.js
 dist/**/*.js
+assets/animations
+android
+ios
+vendor
```

Comment:
Is there a reason that `node_modules` shouldn't be added to this?

Reply:
Prettier ignores `node_modules` by default.